src/VuetifyAlgoliaPlaces.vue

```diff
@@ -2,24 +2,26 @@
   <v-autocomplete
     v-model="place"
     v-bind="$attrs"
-    v-on="$listeners"
     :items="places"
     :loading="loading"
     :search-input.sync="query"
     :filter="filter"
     return-object
     item-text="value"
     :append-icon="appendIcon"
+    v-on="$listeners"
     @input="onInput"
     @click:clear="onClear"
   >
-    <template slot="item" slot-scope="data">
-      <template v-if="typeof data.item !== 'object'">
-        <v-list-tile-content>{{ data.item }}</v-list-tile-content>
+    <template v-slot:item="{ item }">
```

Comment:
Hum, for now I think we will keep the old version. I don't want to have this package only compatible with Vue 2.6+. 

But after/when merging #314 we will go for a Vue 2.6+ version.